traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -701,16 +691,18 @@ def get_cdns_domains(self):
 	#
 	# CDN Topology
 	#
-	@api_request(u'get', u'cdns/{cdn_name:s}/configs', (u'1.2', u'1.3',))
```

Comment:
This might be a typo for `/api/$version/cdns/configs"`

Reply:
no, you don't need to add that to the path

Reply:
That's not what I mean; we have an API endpoint there so the decorator was probably meant to be `cdns/configs`

Reply:
oh. Yeah, you're probably right